developer-docs-site/docs/nodes/ait/node-requirements.md

```diff
@@ -23,18 +23,49 @@ In test mode, you will be running a local network with one single node, and it s
 
 ## Hardware requirements
 
-For running an Aptos node we recommend the following hardware resources:
-
-  - **CPU**: 4 cores (Intel Xeon Skylake or newer).
+For running an Aptos **Validator node** we recommend the following hardware resources:
+
+  - **CPU**:
+      - 8 cores, 16 threads 
+      - 2.8GHz, or faster
+      - Intel Xeon Skylake or newer
+  - **Memory**: 32GiB RAM.
+
+Example machine types on various clouds:
+  - AWS
+      - c5.4xlarge
+      - c6i.4xlarge
+  - GCP
+      - c2-standard-16
+  - Azure
+      - Standard_B8ms
+
+For running an Aptos **Fullnode** we recommend the following hardware resources:
+
+  - **CPU**:
+      - 4 cores, 8 threads 
+      - 2.8GHz, or faster
+      - Intel Xeon Skylake or newer
   - **Memory**: 8GiB RAM.
 
+Example machine types on various clouds:
+  - AWS
+      - c5.2xlarge
+      - c6i.2xlarge
+  - GCP
+      - c2-standard-8
+  - Azure
+      - Standard_B4ms
+
 ## Storage requirements
 
 The amount of data stored by the Aptos Blockhain depends on the ledger history (length) of the blockchain and the number of on-chain states (e.g., accounts). These values depend on several factors, including: the age of the blockchain, the average transaction rate and the configuration of the ledger pruner.
 
 We recommend nodes have at least 300GB of disk space to ensure adequate storage space for load testing. You have the option to start with a smaller size and adjust based upon demands. You will be responsible for monitoring your node's disk usage and adjusting appropriately to ensure node uptime.
```

Comment:
We should also ask IOPS for disk.

Reply:
In general recent years local SSD should be good enough. If people choose to use remote one on cloud, they need to:
1. Buy enough IOPS (number TBD, I feel asking for 20K-30K is not crazy, however on AWS the max of gp2/gp3 is capped at 16K, and io1/io2 is ~10x more expensive)
2. Make sure their instance type can support enough IOPS.
3. Make sure the bandwidth is enough (DB is able to consume 300M bytes per second when there is enough load)